ngi_reports/stockholm/project_summary.py

```diff
@@ -36,6 +37,13 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 self.LOG.error("No project name found - please specify using '--project'")
                 raise KeyError("No project name found - please specify using '--project'")
         
+        ## Check and exit if signature not provided
+        if not kwargs.get('signature'):
+            self.LOG.error("It is required to provide Signature/Name generating 'project_summary' report")
```

Comment:
For the future - there's a logging level called `exception` which may be better here: https://docs.python.org/2/library/logging.html#logging.Logger.exception

(or `critical` at least)